company-profiles/cosmic-chimps.md

```diff
@@ -0,0 +1,33 @@
+# Cosmic Chimps
+
+## Company blurb
+
+Design strategies with Cloud and DevSecOps mainly for Startups companies.
+
+## Remote status
+
+You'll work 100% remotely independient of your location.
+
+## Region
+
+Spain and México
+
+## Company technologies
+
+- Java
+- Flutter
+- PHP
+- Amazon Web Services
+- Jira
+- Git
+- Docker
+- Ansible
+
+## Office locations
+
+Pla de Pau Vila, Barcelona, Spain
+Liverpool 174, Juárez, Cuahutemoc, CDMX, México
+
+## How to apply
+
+No openings currently
```

Comment:
Please link to a careers page so that when openings are availble, they can be found.

Reply:
There is no a careers page but I link to the jobs site on the company profile to Linkedin.